pkg/api/testing/fuzzer.go

```diff
@@ -382,11 +382,16 @@ func FuzzerFor(t *testing.T, version schema.GroupVersion, src rand.Source) *fuzz
 			}
 			if c.RandBool() {
 				c.Fuzz(&ev.ConfigMapRef)
+			} else if c.RandBool() {
```

Comment:
Should this be a separate if statement?
The code just looks confusing when testing the same thing in an 
if c.RandBool(){} else if c.RandBool(){}